metricbeat/module/kubernetes/state_node/_meta/data.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@timestamp": "2017-05-04T17:47:20.829Z",
+  "beat": {
+    "hostname": "X1",
+    "name": "X1",
+    "version": "6.0.0-alpha1"
+  },
+  "kubernetes": {
+    "node": {
+      "allocatable": {
```

Comment:
I've adapted the code to use mb.Namespace (really cool!) but I still don't see the need of adding `state` to the metric key. Let's talk about this tomorrow